terraform/groups/ecs-service/locals.tf

```diff
@@ -7,6 +7,8 @@ locals {
   container_port              = "8080"
   docker_repo                 = "filing-history-delta-consumer"
   kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 10
+  lb_listener_paths           = ["/filing-history-delta-consumer/*"]
```

Comment:
this needs to be `"/company/*/filing-history*" `I think for load balancer listener paths I was talking to Billy about it before christmas for this, that was mainly focused on the api though so I'm happy to push this through and just see if it breaks

Reply:
That path is only relevant to the data API so what we've got should be fine for the consumer.

Reply:
chatted to chris about it